home/lncm/.lnd/lnd.conf

```diff
@@ -0,0 +1,37 @@
+[Application Options]
+
+datadir=/home/lncm/.lnd/data
+logdir=/home/lncm/.lnd/logs
+maxlogfiles=3
+maxlogfilesize=10
+
+tlscertpath=/home/lncm/.lnd/tls.cert
+tlskeypath=/home/lncm/.lnd/tls.key
+noseedbackup=1
```

Comment:
We need to remove this option, since it prevents seed backups. It can only be used on testnet safely

Reply:
Done